### PR TITLE
fix(config): change scenes sent config and on/off mode - fibaro walli dimmer

### PR DESCRIPTION
**Pull request by Micka41**
fixe : #2424

### CODE REVIEW

**Comment by Micka41** [comment]
@zwave-js-assistant update

File: packages/config/config/devices/0x010f/fgwdeu.json [hunk 1]
```diff
@@ -272,45 +262,23 @@
 		},
 		"40": {
 			"label": "First button – scenes sent",
-			"description": "determines which actions result in sending scene IDs",
+			"description": "determines which actions result in sending scene IDs (0 - no scenes; 1 - Key pressed once; 2 - Key pressed twice; 4 - Key pressed 3 times; 8 - Key held down and released)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgwdeu.json:265]: This should be modeled as [partial parameters](https://zwave-js.github.io/node-zwave-js/#/config-files/file-format?id=partial-parameters)

File: packages/config/config/devices/0x010f/fgwdeu.json [hunk 2]
```diff
@@ -272,45 +262,23 @@
...
-					"label": "Key held down and released",
-					"value": 8
-				}
-			]
+			"allowManualEntry": true
 		},
 		"41": {
 			"label": "Second button – scenes sent",
-			"description": "determines which actions result in sending scene IDs",
+			"description": "determines which actions result in sending scene IDs (0 - no scenes; 1 - Key pressed once; 2 - Key pressed twice; 4 - Key pressed 3 times; 8 - Key held down and released)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x010f/fgwdeu.json:276]: dito

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks! Unfortunately this device file seems to come from an earlier automated import from other projects and contains some more errors and/or needs cleaning up.

Since you touched it, we should make sure it is up to our standards. Please read the Style Guide and make the necessary corrections where necessary.

If you have any questions, @blhoward2 and I will gladly assist you.

**Comment by AlCalzone** [comment]
@Micka41 are you still interested in getting this merged?

**Comment by Micka41** [comment]
> @Micka41 are you still interested in getting this merged?

Yes, maybe today

**Pull request closed without merging**